edx_api_doc_tools/conf_utils.py

```diff
@@ -0,0 +1,148 @@
+"""
+Functions for setting up API doc generation & viewing in a repository.
+
+External users: import these from __init__.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.conf import settings
+from django.conf.urls import url
+from django.views.generic.base import RedirectView
+from drf_yasg import openapi
+from drf_yasg.generators import OpenAPISchemaGenerator
+from drf_yasg.views import get_schema_view
+from rest_framework import permissions
+
+
+def make_docs_urls(api_info):
+    """
+    Convenience function for creating API doc views given an API info object.
```

Comment:
This docstring could use an example of how this function is used, how is it helpful? Ditto for the other functions in this module.

Reply:
Added some examples.